parallel/pom.xml

```diff
@@ -57,12 +57,12 @@
         <dependency>
             <groupId>org.apache.jmeter</groupId>
             <artifactId>ApacheJMeter_http</artifactId>
-            <version>3.1</version>
+            <version>5.6.3</version>
         </dependency>
         <dependency>
             <groupId>kg.apc</groupId>
             <artifactId>jmeter-plugins-cmn-jmeter</artifactId>
-            <version>0.3</version>
```

Comment:
Are tests running properly after these changes?

As far as I could see the kg.apc - jmeter commons dependency isn't fully compatible with newer versions of JMeter. Since the mentioned dependency is linked to an older version of JMeter, do you relate?

Reply:
@jloisel I just ran the tests and it seems that broke the test stage part of the maven lifecycle. Would you be willing to address it?

Reply:
I'm not sure what can be done here. The latest cmn plugin version is 0.7 from 2021:
https://mvnrepository.com/artifact/kg.apc/jmeter-plugins-cmn-jmeter

I guess you could try to use this version instead of 0.7. 0.7 is also linked to an older version of jmeter (2.13), not sure why this plugin requires jmeter cmn.

Reply:
People are using the plugin without any issues on JMeter 5.6.x so I guess it's a non-issue.

Reply:
> I guess you could try to use this version instead of 0.7. 0.7 is also linked to an older version of jmeter (2.13), not sure why this plugin requires jmeter cmn.

The plugin requires the jmeter cmn in order to be tested. The jmeter common and jmeter emulator dependencies are used to set up a JMeter environment when required. A JMeter environment is required when a plugin interacts with the UI of JMeter or the actual Core or JMeter like the case of the Parallel Controller plugin.

Please for instance, see [ParallelSamplerTest.java](https://github.com/Blazemeter/jmeter-bzm-plugins/blob/d06a552f248140b4543dbb92e4434cca16e032cf/parallel/src/test/java/com/blazemeter/jmeter/http/ParallelHTTPSamplerTest.java#L23) in order to verify usage of that dependency.

Reply:
> People are using the plugin without any issues on JMeter 5.6.x so I guess it's a non-issue.

It is important to ensure that tests are running and passing, as they serve as protection against potential problems in the future.

You may want to reiterate the importance of maintaining test coverage and how it helps ensure the stability and reliability of the plugin regardless of current usage in JMeter 5.6.x. You can emphasize the value of passing tests not only for the current version but also for future compatibility and overall quality assurance.

Reply:
Expect contact from my side in a couple of hours

Reply:
Thanks for your insight. I fixed the issue by reverting the dependencies to the original version. The tests should be running fine now!

Reply:
Hey, thank you so much for fixing it!

Reply:
I did some QA on the changes and it works for older and newer versions of JMeter 💪